contracts/PartnerConfiguration/PartnerConfiguration.sol

```diff
@@ -231,7 +238,11 @@ contract PartnerConfiguration is IPartnerConfiguration, Ownable {
         emit FeePercentageChanged(preModifiedValue, feePercentage);
 
         if (preModifiedValue == feePercentage) {
-            revert("Param being modified is same as new param");
+            revert(_UN_NECESSARY_MODIFICATION_ERROR_MSG);
+        }
+
+        if (feePercentage > _PERCENT100_WITH_PRECISION18) {
```

Comment:
Should we add upper threshold to the fee percentage? For example, it can only be set a value between %0 and %10. Otherwise, this can be prone to abuse by partners.

Reply:
Nah, partner configuration values can only be set by us.